### PR TITLE
Add AMBR parameters in entrypoint.sh and data_provisionning_users scr…

### PR DESCRIPTION
**Pull request by lionelgo**
Add AMBR parameters in entrypoint.sh and data_provisionning_users scripts
Add missing Extended-***-***-BW-*L AVPs
Enable build on ubuntu 20.04
Add wait-for-it.sh in entrypoint

Signed-off-by: lionelgo <29477918+lionelgo@users.noreply.github.com>

### CODE REVIEW

File: build/tools/build_helper.fd [hunk 1]
```diff
@@ -415,7 +415,13 @@ install_opencoord.org_freediameter() {
     if ! patch -R -p1 -s -f --dry-run <  $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch; then
           patch -p1 <  $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch
     else
-      echo_info "Detected freeDiameter $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch alredy applied"
+      echo_info "Detected freeDiameter $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch already applied"
```
Comment (rdefosse) [inline build/tools/build_helper.fd:418]: @lionelgo 

This new patch is also applied on the RHEL8 builds? Just asking

File: docker/Dockerfile.rhel8-2.oc4-4 [hunk 2]
```diff
@@ -55,6 +55,10 @@ RUN rm /etc/rhsm-host && \
     echo "/usr/local/lib" > /etc/ld.so.conf.d/local-lib.conf && \
     echo "/usr/local/lib64" >> /etc/ld.so.conf.d/local-lib.conf
 
+# wait for it
```
Comment (rdefosse) [inline docker/Dockerfile.rhel8-2.oc4-4:58]: @lionelgo 

can you please also apply the same changes to : 

- `docker/Dockerfile.rhel8` file ---> It is the one used by CI
- `docker/Dockerfile.centos7/8` files  (do we want to keep them?)

File: scripts/entrypoint.sh [hunk 3]
```diff
@@ -36,7 +44,8 @@ mkdir -p /openair-hss/logs
 
 # Create the certificates
 pushd /openair-hss/scripts
-./data_provisioning_users --apn ${APN1} --apn2 ${APN2} --key ${LTE_K} --imsi-first ${FIRST_IMSI} --msisdn-first 00000001 --mme-identity oai-mme.${REALM} --no-of-users ${NB_USERS} --realm ${REALM} --truncate False --verbose True --cassandra-cluster ${cassandra_Server_IP}
+./wait-for-it.sh  ${cassandra_Server_IP}:9042
```
Comment (rdefosse) [inline scripts/entrypoint.sh:47]: @lionelgo 

can you please add a comment on the reason for port `9042`?

**Review by rdefosse — CHANGES_REQUESTED** [review]

File: build/tools/build_helper.fd (hunk 1 above)
Comment (lionelgo) [reply build/tools/build_helper.fd:418]: Yes, this for all distribs: 
in init_submodules()
....
  build_freeDiameter
  ret=$?;[[ $ret -ne 0 ]] && return $ret
  build_c_ares
  ret=$?;[[ $ret -ne 0 ]] && return $ret
  build_cpp_driver
  ret=$?;[[ $ret -ne 0 ]] && return $ret

called by check_install_hss_software()
called by build_hss_rel14 script

File: docker/Dockerfile.rhel8-2.oc4-4 (hunk 2 above)
Comment (lionelgo) [reply docker/Dockerfile.rhel8-2.oc4-4:58]: Done, we can keep Dockerfile.centos8  for the time being.

File: scripts/entrypoint.sh (hunk 3 above)
Comment (lionelgo) [reply scripts/entrypoint.sh:47]: It is the serving port for Apache Cassandra native protocol clients

**Review by rdefosse — APPROVED** [review]

**Pull request merged**